crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -449,6 +449,8 @@ impl EvalContext {
                 SyntaxContext::empty(),
             )),
 
+            Expr::Await(AwaitExpr { arg, .. }) => self.eval(arg),
```

Comment:
Technically it's not super correct. e. g. `(await { then: () => ({ v: 42 }), v: 43 }).v` evaluates to 43 instead of 42, but I'm fine with that for now...